### ISSUE TITLE
RC0006: Provide a textual explanation of the Example DID document

### ISSUE THREAD

**Issue by jorritspee** [nuts-foundation/nuts-specification#66]
The Example DID document provides good insight in the syntax of de DID document. For me as a newbie I would like to know: What is the meaning of this Example DID document?
For instance: 
This documents says that "did:nuts:123" is the DID of something.
This identifier can be updated/controled by two entities: "did:nuts:123", "did:nuts:7368245"
The verificationmethod is this and that.
The capabilityInvocation tells us that blabla.

**Comment by woutslakhorst** [comment]
@jorritspee does the example still need explaining with the knowledge from the W3C specification: https://www.w3.org/TR/did-core/

**Comment by stevenvegt** [comment]
I agree we should add a non normative section where we can roughly explain the concepts of DIDs and VCs in relation to the Nuts node.

Note that you do not need to know and fully understand these specifications if you only want to use the nuts node. Therfor we put this on hold.
